example/lib/preview/preview_store.dart

```diff
@@ -53,6 +53,16 @@ abstract class PreviewStoreBase
     this.localTracks = ObservableList.of(videoTracks);
   }
 
+  @override
+  void onPeerUpdate({required HMSPeer peer, required HMSPeerUpdate update}) {
+    // TODO: implement onPeerUpdate
```

Comment:
update on UI similar to this
![Uploading Screenshot 2022-03-04 at 1.34.13 PM.png…]()